src/components/Dropdown/DropdownTrigger.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+interface DropdownTriggerProps {
+  children: React.ReactNode;
+  className?: string;
+}
+
+/**
+ * Generates a `<summary>` element, styled as a button,
+ * to use within a `Dropdown` component,
+ */
+export const DropdownTrigger = ({
```

Comment:
As you noted elsewhere, this puts the onus on the consuming code to ensure the first child of a `<Dropdown>` is this `<DropdownTrigger>`, and is difficult to enforce - or at least, I'm not sure how to do so right now.

The stories all use basic text as the only children of a `<DropdownTrigger>`; do you intend to use anything else in production? How would you feel about inlining `<DropdownTrigger>` into `<Dropdown>` by adding a `title` prop (or `summary`, or `name`, or `trigger`, or...)?

Reply:
Oof, I had forgotten the requirement that the first element of `<details>` must be `<summary>`, which means yes there is an unstated order requirement here. :/

So I do intend for `DropdownTrigger` to allow multiple children, e.g. an icon + text. Okay actually that's my only example so far. But I think your solution is still a good one. In fact, it may let me condense the hierarchy to simply:

```jsx
<Dropdown trigger={}>
  /* Dropdown items here */
</Dropdown>
```

Then there's no need for `DropdownTrigger` *or* `DropdownMenu` components. I kind of like this! What do you think?

Reply:
That looks great! Passing a component as a prop is a pattern I've seen before, particularly with react-router, so that would have been my next suggestion.

Reply:
@jasonaowen Okay, updated! Ready for you check out again.